app/controllers/documents_controller.rb

```diff
@@ -158,7 +158,13 @@ def fed_hub_request
       @verification_type.add_type_history_element(action: "Hub Request Failed",
                                                   modifier: "System",
                                                   update_reason: "#{@verification_type.type_name} Request Failed due to #{message}")
-      ::Operations::Eligibilities::BuildFamilyDetermination.new.call(family: @person.primary_family, effective_date: TimeKeeper.date_of_record)
+      if @person.primary_family
```

Comment:
@hKalesnikavaideacrew could we just loop though families instead of doing if/else with primary?